cdk/PgStacInfra.ts

```diff
@@ -377,14 +378,14 @@ export class PgStacInfra extends Stack {
       {
         defaultBehavior: { origin: new origins.S3Origin(stacBrowserBucket) },
         defaultRootObject: rootPath,
-        domainNames: [props.stacBrowserCustomDomainName],
+        domainNames: [stacBrowserConfig.customDomainName],
         certificate: acm.Certificate.fromCertificateArn(
           this,
           "stacBrowserCustomDomainNameCertificate",
-          props.stacBrowserCertificateArn,
+          stacBrowserConfig.certificateArn,
```

Comment:
would we want to default to using the main cert unless this is defined in stacBrowserConfig? or just use the main cert?

Reply:
I think they have to be different because the cloudfront cert needs to be in us-east-1

Reply:
oh, right forgot about that portion :+1: